packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.spec.ts

```diff
@@ -89,19 +90,93 @@ function buildTestPerson(): readonly [SchemaDataAndPolicy, PersonType] {
 }
 
 describe("editable-tree", () => {
-    it("proxified forest", () => {
+    it("can use `Object.keys` and `getOwnPropertyDescriptor` with EditableTree", () => {
         const [, proxy] = buildTestPerson();
-        assert.ok(proxy);
+        assert(isUnwrappedNode(proxy));
+
         assert.equal(Object.keys(proxy).length, 5);
-        assert.deepEqual(proxy[getTypeSymbol](undefined, false), personSchema);
-        assert.deepEqual(proxy.address[getTypeSymbol](undefined, false), addressSchema);
+        assert.deepEqual([...Object.keys(proxy)], ["name", "age", "salary", "friends", "address"]);
+
+        // `getOwnPropertyDescriptor` unwraps fields
+        for (const key of Object.keys(proxy)) {
+            const fieldKey: FieldKey = brand(key);
+            const descriptor = Object.getOwnPropertyDescriptor(proxy, key);
+            assert(descriptor !== undefined);
+            let expected = proxy[fieldKey];
+            if (isUnwrappedNode(descriptor.value)) {
+                descriptor.value = clone(descriptor.value);
```

Comment:
What is the purpose of this assignment?

Reply:
It has no special purpose. I was using it for debugging. It might actually be helpful, but if you think it's confusing, we can safely remove this block.

Reply:
Ah, was this because the original `descriptor.value` data is hidden behind code?
I think a comment would be nice, just to clarify why it is helpful.

Reply:
Yes, that's correct. Done.